Week3/homework/3-step3.js

```diff
@@ -2,23 +2,43 @@
 
 // use a 'for' loop
 function repeatStringNumTimesWithFor(str, num) {
-  // add your code here
+  if (num < 1) {
+    return "";
+  }
+
+  let myStr = str;
+  for (let i = 1; i < num; i++) {
+    str += myStr;
+  }
   return str;
 }
 
 console.log('for', repeatStringNumTimesWithFor('abc', 3));
 
 // use a 'while' loop
 function repeatStringNumTimesWithWhile(str, num) {
-  // add your code here
+  if (num < 1) { return "";}
```

Comment:
You should space the code out properly, this helps in making the code look cleaner and improves readability.